apps/design-system/src/subjects/views/secrets/secrets.tsx

```diff
@@ -121,18 +121,26 @@ export const SecretsPage = ({
   }
 
   return (
-    <Drawer.Root open={isDrawerOpen} onOpenChange={setIsDrawerOpen} direction="right">
-      <Drawer.Content>
-        <Drawer.Header>
-          <Drawer.Title className="text-3xl">Secrets</Drawer.Title>
-          <Drawer.Close onClick={() => setIsDrawerOpen(false)} />
-        </Drawer.Header>
-        <Spacer size={5} />
+    <>
+      <Drawer.Root open={isDrawerOpen} onOpenChange={setIsDrawerOpen} direction="right">
+        <Drawer.Content>
+          <Drawer.Header>
+            <Drawer.Title className="text-3xl">Secrets</Drawer.Title>
+            <Drawer.Close onClick={() => setIsDrawerOpen(false)} />
+          </Drawer.Header>
+          <Spacer size={5} />
 
-        <SecretsHeader onChange={setSelectedType} selectedType={selectedType} />
-        <Spacer size={5} />
-        {renderSecretContent()}
-      </Drawer.Content>
-    </Drawer.Root>
+          <SecretsHeader onChange={setSelectedType} selectedType={selectedType} />
+          <Spacer size={5} />
+          {renderSecretContent()}
+        </Drawer.Content>
+      </Drawer.Root>
+      <ConnectorsRefPage
```

Comment:
i think this should just be rendered inside CreateSecret directly. no need to bring that state out.